nnvm/src/top/tensor/elemwise.cc

```diff
@@ -768,6 +768,108 @@ NNVM_REGISTER_ELEMWISE_BINARY_SCALAR(__rpow_scalar__)
     };
 });
 
+NNVM_REGISTER_ELEMWISE_BINARY_SCALAR(__equal_scalar__)
```

Comment:
We should not introduce scalar ops. Instead, let us directly use equal + constant

Reply:
Sorry, I thought it was a relay change. Is it possible to omit the nnvm changes for now, given that we are transitioning to relay

Reply:
OK. I can remove nnvm part. The downside for now is that user can't autotune gluoncv model before relay is fully supported by autotvm.

Reply:
As far as I know @merrymercy and @eqy already add autotvm trask extraction support?

Reply:
https://docs.tvm.ai/tutorials/autotvm/tune_nnvm_cuda.html Looks like tutorial not updated？